nix/devtools.nix

```diff
@@ -72,6 +72,9 @@ let
       }
       ''
         ${cabal-install}/bin/cabal v2-clean
+        # clean old coverage data, too
+        rm -rf .hpc coverage
+        find . -type f -iname "*.tix" -delete
```

Comment:
This is probably no longer needed with the temp dir?

Reply:
Correct, the tix files don't need to be deleted. I'll leave the `rm -rf .hpc coverage` in, though. Those still exist.